### DISCUSSION TITLE
61%  building 604/704 modules 100 active ...

### DISCUSSION THREAD

**Discussion by flyingDandelionYD** [webpack/webpack#15731] [General]
**What is the current behavior?**
run dev ，it just shut down the terminal ,the last log is:  61%  building 604/704 modules 100 active ...

1.when I  remove some other subpackages, it just works well.
2. Maybe , the subpackage is larger(one is 300kb+ , and the other is  900kb+)
3. Do not know why ~ ~

**Other relevant information:**
webpack version:  webpack@4.46.0
Node.js version: node - v14.3.0
Operating System: macOS Catalina 10.15.7
Additional tools: Megalo

**Comment by flyingDandelionYD** [comment]
When I run dev , the output  js bundle is less than 3M ,
When failed  >  the js bundle is out of range 3M???

**Comment by vankop** [comment]
please create a reproducible repo
